lib/syntax_tools/src/erl_syntax_lib.erl

```diff
@@ -23,9 +23,6 @@
 %%
 %% This module contains utility functions for working with the
 %% abstract data type defined in the module {@link erl_syntax}.
-%%
-%% @type syntaxTree() = erl_syntax:syntaxTree(). An abstract syntax
```

Comment:
I mean, add a <code>-type syntaxTree() :: ...</code> instead of this edoc type spec.